toplevel/opttoploop.ml

```diff
@@ -336,7 +336,7 @@ let execute_phrase print_outcome ppf phr =
       if !Clflags.dump_typedtree then Printtyped.implementation ppf str;
       let sg' = Typemod.Signature_names.simplify newenv names sg in
       (* Why is this done? *)
-      ignore (Includemod.signatures oldenv sg sg');
+      ignore (Includemod.signatures oldenv ~mark:Mark_positive sg sg');
```

Comment:
It sounds like a good time to remove the comment above. We could maybe add a comment that we are emitting a warning for toplevel phrase like:
```OCaml
let x = 0 let x = 1;;
```